### ISSUE TITLE
Scrolltotop do not remember settings.

### ISSUE THREAD

**Issue by hilandae** [pratikabu/scrolltotop#34]
I love this addon but after every update the settings reset to default and i have every time to change them back to my favor.

**Comment by pratikabu** [comment]
hello, sad to hear that. which browser version are you using.
Can you give me exact version detail.

Thanks

**Comment by hilandae** [comment]
Hello,
I have Firefox 34.0 in Greek language.

Best Regards

2015-01-19 15:06 GMT+02:00 Pratik Soni <notifications@github.com>:

**Comment by pratikabu** [comment]
have you re-installed it. Also, firefox 35 is available, can you update it and test once.
Best Regards
Pratik

**Comment by hilandae** [comment]
No i haven't but the same happens on two pc. One home and one to work both with latest Firefox version and windows 7. I update Firefox regularly. I will try to uninstall scrolltotop and install again to see if it fix the problem but i think that scrolltotop resets the settings after a new update released  and renew my pc.

**Comment by Akisl** [comment]
Same with me. Every time after restart or logoff brings me to the settings page and i have to set settings again. Windows 7 Firefox 35.0.1.

**Comment by camolin3** [comment]
Same issue. Using Firefox 37.0.2 on Ubuntu 15.04 (maybe it is already fixed but I won't know until a new update of scrolltotop).

**Comment by pratikabu** [comment]
hello all, first of all its really sad to hear about the problem you all are facing. To replicate it, I tried upgrading from FF 35 to 36 and then to 37. It worked smoothly and I didn't face any setting reset issue. I'm using Scroll To Top 4.5.1.

Since FF 35, one of the API was changed and because of that Scroll To Top 4.5 was not saving the settings. But I provided the fix in version 4.5.1 and after that I didn't face any issues.

Request you guys to use Scroll To Top 4.5.1.

Thanks
